CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+### citus v10.0.2 (March 3, 2021) ###
+
+* Fixes insert query with CTEs/sublinks/subqueries etc
+
+* Removes limits around long table names
+
+* Uses `PROCESS_UTILITY_QUERY` in utility calls
```

Comment:
We should do as we did in v9.5.4:
```suggestion
* Fixes a compatibility issue with pg_audit in utility calls
```

Sorry, I should have updated that related pr's description